src/main/java/tc/oc/pgm/map/MapTagImpl.java

```diff
@@ -0,0 +1,57 @@
+package tc.oc.pgm.map;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import tc.oc.pgm.api.map.MapTag;
+
+public class MapTagImpl implements MapTag {
+
+  private static final String HASHTAG = "#";
```

Comment:
I called this `SYMBOL` originally because all these different special characters (maily on scoreboard) were called `SYMBOL`s.